SREP-1186: Add stub for configure-alertmanager-operator

lisa opened the pull request:
Note that the hash length and template file need to make their way to
the operator's git repo (the favoured hash length is 8 characters, and
the template does not exist yet).

Signed-off-by: Lisa Seelye <lseelye@redhat.com>

lisa committed:
SREP-1186: Add stub for configure-alertmanager-operator

Note that the hash length and template file need to make their way to
the operator's git repo (the favoured hash length is 8 characters, and
the template does not exist yet).

Signed-off-by: Lisa Seelye <lseelye@redhat.com>

lisa commented:
/cc @jharrington22 
/cc @rogbas

jharrington22 commented on configure-alertmanager-operator/configure-alertmanager-operator.yaml:
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: configure-alertmanager-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```
We would want to change the name of this to sss-artifacts-template.yaml ?

lisa replied:
Maybe? I thought about it, but wanted to keep it consistent.

jmelis replied:
fwiw we don't have a preference from the app-sre side of things

jmelis commented:
Also, don't forget to add a new context to https://github.com/app-sre/saas-hive/blob/master/config.yaml

jharrington22 commented:
My fault, I did not mention the context specifically to @lisa :)

lisa committed:
SREP-1186: Add configure-alertmanager-operator to context

Signed-off-by: Lisa Seelye <lseelye@redhat.com>

lisa commented:
> Also, don't forget to add a new context to https://github.com/app-sre/saas-hive/blob/master/config.yaml

Done!

lisa committed:
SREP-1186: normalize directory name

Signed-off-by: Lisa Seelye <lseelye@redhat.com>

jharrington22 reviewed (approved):
LGTM - lets wait for @jmelis to merge when ready.

jmelis commented on configure-alertmanager-operator-services/configure-alertmanager-operator.yaml:
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: configure-alertmanager-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```
this path doesn't exist yet, right?

lisa replied:
Is that in https://github.com/openshift/configure-alertmanager-operator ?

lisa replied:
It is, working on a PR.

lisa replied:
@jmelis https://github.com/openshift/configure-alertmanager-operator/pull/16 was merged, can you have another look here?

lisa committed:
Address path issue

Pairs with https://github.com/openshift/configure-alertmanager-operator/pull/16

jharrington22 committed:
Merge branch 'master' into srep-1186-add-configure-alertmanager-operator-stub

jharrington22 commented:
LGTM - merged master to update branch

jharrington22 reviewed (approved):
Checks passed.
